source/DSCResources/Resources/Vsphere.VmHostVssSecurity.ps1

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+$rules = $stig.RuleList | Select-Rule -Type 'VsphereVssSecurityRule'
+
+foreach ($virtualStandardSwitch in $virtualStandardSwitchGroup)
+{
+    foreach ($rule in $rules)
+    {
+        if ($rule.AllowPromiscuous)
+        {
+            $allowPromiscuous = $rule.AllowPromiscuous
+        }
+        if ($rule.ForgedTransmits)
+        {
+            $forgedTransmits = $rule.ForgedTransmits
+        }
+        if ($rule.MacChanges)
+        {
+            $macChanges = $rule.MacChanges
+        }
+
+        $idValue += $rule.id
+    }
+
+    VmHostVssSecurity "$virtualStandardSwitch-$idValue"
+    {
+        Name             = $HostIP
+        Server           = $ServerIP
+        Credential       = $Credential
+        VssName          = $VirtualStandardSwitch
+        AllowPromiscuous = [bool] $allowPromiscuous
```

Comment:
Same in as the previous file can we do:
VmHostVssSecurity foo
{
    AllPromiscuous = [bool]$rule.AllowPromiscuous
}

Reply:
same reply as above, there are two foreach loops going on here.  This particular resource block requires multiple rules to be aggregated, the $rule in this case is not available within the $virtualStandardSwitchGroup loop where the resource block resides.